kogito-cloud-operator.jenkinsfile

```diff
@@ -274,4 +284,17 @@ void installOpenshiftClient(){
         tar xzf ./\${PACKAGE_NAME}
         mv oc \${OSE_CLIENT_DIR}
     """
+}
+
+void installMaven(){
+    sh """       
+        rm -rf \$HOME/bin/maven
+        RELEASE_VERSION=3.6.3
+        curl -LO https://mirrors.ircam.fr/pub/apache/maven/maven-3/\${RELEASE_VERSION}/binaries/apache-maven-\${RELEASE_VERSION}-bin.tar.gz
```

Comment:
Maybe would be good to use a US mirror for faster download?

Reply:
good point. updated :)